storages/python/python-storage-plugins/postgresql.py

```diff
@@ -191,7 +191,7 @@ def table_create(self, meta, dims=None):
         ]
         for p in meta["properties"]:
             decl = f'"{p.name}" {to_pgtype(p.type)}'
-            if len(p.dims):
```

Comment:
This change does not make sense to me. You test for the length of shape, but use dims, as before you tested and used dims

Reply:
Sell seen. p.dims  is an backward-compatible alias for p.shape. We should use p.shape